packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -273,8 +305,12 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
   /**
    * Helper function for makeUpdateCommit() to allow unit tests to update the commit before it is signed.
    */
-  private static _makeRawCommit<T>(prev: CommitID, oldContent: T, newContent: T | null): RawCommit {
-    const patch = jsonpatch.compare(oldContent, newContent || {})
+  private static _makeRawCommit<T>(
+    prev: CommitID,
+    oldContent: T | null,
+    newContent: T | null
+  ): RawCommit {
+    const patch = jsonpatch.compare(oldContent ?? {}, newContent ?? {})
```

Comment:
is the use of jsonpatch entirely new or just moved here?

Reply:
Just moved, we were already using it